doc/visual-programming/source/widgets/data/pivot.md

```diff
@@ -0,0 +1,68 @@
+Pivot Table
+===========
+
+Reshape data table based on column values.
+
+**Inputs**
+
+- Data: input data set
+
+**Outputs**
+
+- Pivot Table: contingency matrix as set in the widget
+- Filtered Data: subset selected from the plot
+- Grouped Data: data table grouped by row values
+
+**Pivot Table** summarizes the data of a more extensive table into a table of statistics. The statistics can include sums, averages, counts, etc. The widget also allows selecting a subset from the plot and grouping by row values, which have to be a discrete variable. Data with only numeric variables cannot be displayed in the plot.
+
+![](images/Pivot-stamped.png)
+
+1. Discrete or numeric variable that will be used for row values. Numeric variables are considered as integers in this case. Variable values will appear as rows in the table.
```

Comment:
The last sentence is perhaps redundant. You can also remove `"in this case."` Also perhaps `that will be`.